### PR TITLE
Add: basic frame for homepage and watch later page

### PR DESCRIPTION
**Pull request by varun13169**
Add a basic frame for the homepage and watch the later page
Ignore data and coloring

image

image

### CODE REVIEW

File: src/components/Card/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
+import "./card.css";
+import { BinSVG, WishlistHeartSVG } from "../../assets/svgReactComponents";
+import { Link, useLocation } from "react-router-dom";
+// import { useCart, useWishlist } from "../../contexts";
```
Comment (DivyanshSareen) [inline src/components/Card/Card.jsx]: remove comment if not for future use

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      qty: 1,
+    },
+    priAction: { name: "Watch Later", action: () => {} },
+    secAction: () => {},
+    wishlistAction: () => {},
+  };
+  const { itemDetails, priAction, secAction, wishlistAction } = itemCardData;
+
+  //   const { cart, setCart } = useCart();
+  //   const { wishlist, setWishlist } = useWishlist();
```
Comment (DivyanshSareen) [inline src/components/Card/Card.jsx]: remove comments

File: src/components/Card/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        )}
+      </div>
+
+      <div className="dui-card-prod-hzntl__info">
+        {/* <p className="dui-card-prod-hzntl__secondary-text">{productName}</p> */}
+        <h3 className="dui-card-prod-hzntl__primary-text dui-util-fw-blk dui-util-txt-align-left">
+          ${salePrice}{" "}
+          {/* <s className="dui-card-prod-hzntl__secondary-text dui-util-txt-sm">
+            ${originalPrice}
+          </s> */}
```
Comment (DivyanshSareen) [inline src/components/Card/Card.jsx]: remove comments in this file

**Review by DivyanshSareen — APPROVED** [review]
The folder structure and files are nicely put together. The abstractions on assets and components through index.js make the code easy to read and understand. Good to merge.

File: src/components/Carousal/carousal_.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+const slides = document.querySelectorAll(".slideshow-container__slide");
+const dots = document.querySelectorAll(".slideshow-container__navigator");
+const prevAction = document.querySelector(".slideshow-container__action-prev");
+const nextAction = document.querySelector(".slideshow-container__action-next");
+// const slides = document.querySelectorAll()
```
Comment (dev-prabhat) [inline src/components/Carousal/carousal_.js:5]: Why you have used Imperative JS in react component

**Review by itsmebipul00 — APPROVED** [review]
Good to merge

**Pull request merged**